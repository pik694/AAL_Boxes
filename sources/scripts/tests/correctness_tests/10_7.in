10
87 90 44
80 14 39
41 95 41
49 55 85
62 80 9
89 37 9
68 40 51
28 28 86
97 8 23
45 79 14

