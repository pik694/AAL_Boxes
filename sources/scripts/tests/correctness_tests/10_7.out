8 5 1 4 9 3 7 2 6 0 
1125009->869095
